index.html

```diff
@@ -259,6 +270,134 @@ <h3>Video Content Hints</h3>
+        encoding at the configured framerate and resolution, the encoder
+        must make a choice on how to modify the encoding parameters.
+      </p>
+      <p>
+        This section defines terms to describe the choice, and an enum
+        that can be used to indicate that choice in an API.
+      </p>
+      <div>
+        <pre class="idl"
+>enum RTCDegradationPreference {
+  "maintain-framerate",
+  "maintain-resolution",
+  "balanced"
+};</pre>
+        <table data-link-for="RTCDegradationPreference" data-dfn-for=
+        "RTCDegradationPreference" class="simple">
+          <tbody>
+            <tr>
+              <th colspan="2"><code><a>RTCDegradationPreference</a></code> Enumeration description</th>
+            </tr>
+            <tr>
+              <td data-tests="RTCRtpParameters-degradationPreference.html"><dfn data-idl><code>maintain-framerate</code></dfn></td>
+              <td>
+                <p>Degrade resolution in order to maintain framerate.</p>
+              </td>
+            </tr>
+            <tr>
+              <td><dfn data-idl><code>maintain-resolution</code></dfn></td>
+              <td>
+                <p>Degrade framerate in order to maintain resolution.</p>
+              </td>
+            </tr>
+            <tr>
+              <td data-tests="RTCRtpParameters-degradationPreference.html"><dfn data-idl><code>balanced</code></dfn></td>
+              <td>
+                <p>Degrade a balance of framerate and resolution.</p>
+              </td>
+            </tr>
+          </tbody>
+        </table>
+      </div>
+      An attribute is defined for RTCRtpSendParameters that allows this
+      to be explicitly indicated for an RTCRtpSender:
+
+      <pre class='idl'>partial dictionary RTCRtpSendParameters {
+        RTCDegradationPreference degradationPreference;
+       };</pre>
+      <section>
+        <h2>Dictionary <code><a>RTCRtpSendParameters</a></code> New Members</h2>
+        <dl>
+          <dt data-tests="RTCRtpParameters-degradationPreference.html"><dfn data-idl><code>degradationPreference</code></dfn> of type
+            <span class="idlMemberType"><a>RTCDegradationPreference</a></span>,
+            defaulting to <code>"balanced"</code></dt>
+          <dd>
+            <p>When bandwidth is constrained and the
+              <code><a>RTCRtpSender</a></code> needs to choose between degrading
+              resolution or degrading framerate,
+              <code>degradationPreference</code> indicates which is
+              preferred.</p>
+          </dd>
+        </dl>
+      </section>
+    <section>
+      <h3>Behavior of an RTCPeerConnection</h3>
+      <p>An <a>RTCRtpSender</a> transmitting a MediaStreamTrack for which a
+        contentHint attribute has been
+        set MUST use the following degradation preferences, unless an
+        explicit degradationPreference attribute has been set in the sender's
+        parameters:
```

Comment:
s/attribute/member/ or parameter. Should probably reference [setParameters](w3c.github.io/webrtc-pc/#dom-rtcrtpsender-setparameters).